packages/frint-cli/src/providers.mock.ts

```diff
@@ -36,7 +43,7 @@ export default [
     name: 'console',
     cascade: true,
     useFactory: function useFactory() {
-      const fakeConsole = {
+      const fakeConsole: FakeConsole = {
```

Comment:
native `Console` not supported? https://github.com/Microsoft/TypeScript/blob/master/src/lib/dom.generated.d.ts#L2983-L3011